docs/src/modules/flavors.md

```diff
@@ -63,6 +63,22 @@ For devices with "boot-as-recovery", the typical LineageOS flashing process invo
 The `boot.img` and `ota` targets can be built using `nix-build ... -A bootImg` or `nix-build ... -A ota`, respectively.
 Check the upstream documentation for your particular device before following the above instructions.
 
+### Note on signing
+When you build a LineageOS image, the system image is cryptographically signed by default. There are three options concernig the keys the image can be signed with:
+
+- the official LineageOS keys (release-keys). We don't have access to them, so the only images signed with these keys are the official LineageOS builds. If you previously installed LineageOS from official sources, these are the keys you are currently using.
+- the test keys (test-keys). These keys are publicly available, and Robotnix uses them by default.
+- your own release keys. Please see [the relevant section in the Building page](../building.md#building-and-signing-releases).
+
+When you run LineageOS and install apps that save stuff to your phone, that data is "coupled" to the keys your LineageOS install was signed with (Android FBE or FDE works with keys that are protected by the keymaster HAL, whose keys are derived from the build keys). If you try to boot a LineageOS install with an image that was signed with different build keys than the ones your user data was coupled to, it will complain and not boot at best (or you'll loose all your data at worse).
```

Comment:
The parenthesis is from my own research. I hope it is correct.

Reply:
I couldn't figure out the exact link between those keys and the data, so I removed the parenthesis.

The nearest I could find is https://stackoverflow.com/questions/56222616/where-does-android-store-the-platform-shared-media-and-release-keys-in-the-fin, but still it doesn't say how the build keys affects the data and how it is linked to the keys in packages.xml. Well, I guess the specifics are not that important here anyway.